gpii/node_modules/testing/src/CloudBasedOAuth2.js

```diff
@@ -134,9 +134,9 @@ gpii.test.cloudBased.oauth2.verifyAuthorizeResponse = function (body, authorizeR
 
 gpii.test.cloudBased.oauth2.sendLoginRequest = function (loginRequest, options) {
     var formBody = {username: options.username, password: options.password};
-    
+
     formBody = gpii.test.cloudBased.oauth2.filter(formBody, options, "loginRequestForm");
-    
```

Comment:
Yeah,

that'd be great! I usually tend to do a ```git diff <branch>``` when merging pull requests, among other things, it helps me to find this kind of whitespaces that aren't recognized by jshint.

And sorry for not glorifying our incredible elegance of our filtering system, which I love too xD